### DISCUSSION TITLE
EXPIRE and key value

### DISCUSSION THREAD

**Discussion by eviltik** [dragonflydb/dragonfly#138] [General]
Hi,

I can be out of date, but as i know in redis, when you monitor for key expirations, you only get the key, not the value, see https://github.com/redis/redis/issues/1876 and https://stackoverflow.com/questions/18328058/redis-notifications-get-key-and-value-on-expiration 

I never understood why this as never been implemented in redis as an option. My keys have small values (i.e < 500 bytes).

Does dragonflydb provide value of expiring key ? ty

**Comment by romange** [comment]
DF did not implement keyspace notifications yet. I do not know how popular this feature is. When we gather enough feedback from the community we will decide on how to implement it.
